coverage/functions.csv

```diff
@@ -1,3 +1,4 @@
 ActiveUsersService.ts,3
 AuthenticationService.ts,2
```

Comment:
t'sais enlever celui-ci plz ? Supprime ton coverage/src, relance les tests, puis coverage.py generate again

Reply:
$ ./scripts/coverage.py generate
bash: ./scripts/coverage.py: /usr/bin/python: bad interpreter: No such file or directory

et au cas où c'était ça, car ça me semblait linuxisé, j'ai supprimé la première ligne, pas mieux :
$ ./scripts/coverage.py generate
./scripts/coverage.py: line 1: import: command not found
./scripts/coverage.py: line 2: import: command not found
./scripts/coverage.py: line 3: from: command not found
./scripts/coverage.py: line 4: import: command not found
./scripts/coverage.py: line 5: from: command not found
./scripts/coverage.py: line 7: syntax error near unexpected token `sys.argv'
./scripts/coverage.py: line 7: `if len(sys.argv) < 2:'

Reply:
lance `python ./scripts/coverage.py generate` !